testData/solutions/win32.json5

```diff
@@ -1691,396 +1773,434 @@
+                            "type": "string"
                         }
                     },
-                    "Options.SaySelectedFirst": {
+                    "OutputModes.CONTROLTYPE": {
                         "schema": {
-                            "title": "Say Selected First",
-                            "description": "Whether to announce the selected text first.",
-                            "type": "number",
-                            "default": 0,
-                            "enum": [
-                                0,
-                                1
-                            ],
-                            "enumLabels": [
-                                "off",
-                                "on"
-                            ]
+                            "title": "Control Type Verbosity",
+                            "description": "How verbose \"Control Type\" announcements should be.",
+                            "type": "string"
                         }
                     },
-                    "Options.SayStateFirst": {
+                    "OutputModes.DIALOG_NAME": {
                         "schema": {
-                            "title": "Say State First",
-                            "description": "Whether to announce the window state before title/text.",
-                            "type": "number",
-                            "default": 0,
-                            "enum": [
-                                0,
-                                1
-                            ],
-                            "enumLabels": [
-                                "off",
-                                "on"
-                            ]
+                            "title": "Dialog Name Verbosity",
+                            "description": "How verbose \"Dialog Name\" announcements should be.",
+                            "type": "string"
                         }
                     },
-                    "Options.SayWindowTypeFirst": {
+                    "OutputModes.DIALOG_TEXT": {
                         "schema": {
-                            "title": "Say Window Type First",
-                            "description": "Whether to announce the type of window before announcing the window title/text.",
-                            "type": "number",
-                            "default": 0,
-                            "enum": [
-                                0,
-                                1
-                            ],
-                            "enumLabels": [
-                                "off",
-                                "on"
-                            ]
+                            "title": "Dialog Text Verbosity",
+                            "description": "How verbose \"Dialog Text\" announcements should be.",
+                            "type": "string"
```

Comment:
We should not have any catch all settings like this.  We don't even have a single example of what's acceptable (i.e. the default).  IMO this should be fleshed out or commented out.

Reply:
This has been commented out.